modules/stanford_earth_events/src/Form/EventImportersForm.php

```diff
@@ -126,6 +150,31 @@ public function submitForm(array &$form, FormStateInterface $form_state) {
       ->set('source.urls', array_filter($listed))
       ->save();
 
+    foreach (array_merge($listed, $unlisted) as $name) {
```

Comment:
Trixy. This looks great to me. I suppose it is possible that the fetcher wouldn't be able to get the xml contents due to connectivity or some other issue but you would only end up with an empty description.

Reply:
Perhaps you would like to default the description to the url as well?

Reply:
The description has to be something I can map to the Department/Program term so I'm not sure the feed URL will help. At some point we should probably have a manually entered taxonomy that maps the event code in the URL to a more descriptive term, but this will do for now.